activerecord/test/cases/tasks/mysql_rake_test.rb

```diff
@@ -265,20 +265,36 @@ def setup
 
     def test_structure_dump
       filename = "awesome-file.sql"
-      Kernel.expects(:system).with("mysqldump", "--result-file", filename, "--no-data", "--routines", "test-db").returns(true)
+      Kernel.expects(:system)
+        .with("mysqldump", "--result-file", filename, "--no-data", "--routines", "test-db")
+        .returns(true)
```

Comment:
Did anything in this section change or was it just reformatted? If it's just reformatting I'd rather keep it the way it's now. This will keep the patch focused on the actual change, won't mess with `git blame` and will make it easier to backport if need be.